addons/account/models/account_reconcile_model.py

```diff
@@ -24,7 +24,13 @@ def validate_regex(self):
         for record in self:
             if not (record.narration_regex or record.payment_ref_regex):
                 raise ValidationError(_("Please set at least one of the match texts to create a partner mapping."))
-
+            try:
+                if(record.narration_regex):
+                    re.compile(record.narration_regex) 
+                if (record.payment_ref_regex):
+                    re.compile(record.payment_ref_regex) 
+            except:
```

Comment:
Never do bare except. You can catch `re.error`.

Reply:
You are also missing the entry in the pot file; and it would be nice to tell which expression is wrong in the message.

Reply:
@william-andre thanks for your review! 
Could you check if it is ok ?